ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py

```diff
@@ -163,8 +162,14 @@ def _cleanup_dirty_jobs(self):
         for job_id in self.registry_slice.ids():
             j = stripProxy(stripProxy(self.registry_slice(job_id)))
             status = lazyLoadJobStatus(j)
-            if status == 'completing':
-                j.status = 'submitted'
+            if not j.subjobs and status == 'completing':
```

Comment:
Commenting this part resolved the slow loading issue.

Reply:
I see, thanks! I will fix and resubmit that part.